FileShareClient/Models/Attributes.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace UKHO.FileShareClient.Models
+{
+    public class Attributes : IEquatable<Attributes> 
+    {
+        public Attributes(string key = default, string[] values = default)
+        {
+            Key = key;
+            Values = values;
+        }
+
+        [DataMember(Name = "key", EmitDefaultValue = false)]
+        public string Key { get; set; }
+
+        [DataMember(Name = "values", EmitDefaultValue = false)]
+        public string[] Values { get; set; }
+
+        public override string ToString()
+        {
+            var sb = new StringBuilder();
+            sb.Append("class BatchAttributesSearchResponse {\n");
```

Comment:
Please fix the class name here and use `nameof` wherever possible to help keep the printed class and field names consistent with the actual class and field names